SeattleDSA/not911.me
# Add preliminary generator

To test:
1. Install ruby
2. cd into the `generator` directory
3. run `gem install bundle jekyll`
4. run `bundle exec jekyll serve`

Check the site out at `localhost:4000`
